core/arguments.js

```diff
@@ -0,0 +1,46 @@
+var consolec = require('./unsafe/console.js'),
+	options = [
+		{
+			long: '--debug',
+			short: '-d',
+			run: function() {
+				console.warn('Debug mode enabled.');
+				consolec.setDebug(true);
+			}
+		},
+		{
+			long: '--log',
+			short: '-l',
+			run: function() {
+				console.warn('Logging mode enabled.');
+				consolec.setLog(true);
+			}
+		},
+		{
+			long: '--timestamp',
+			short: '-t',
+			run: function() {
+				consolec.setTimestamp(true);
+			}
+		}
+	];
+
+exports.runArguments = function(args) {
+	for (var i = 0; i < args.length; i++) {
+		var arg = args[i],
+			pargs = options.filter(function(value) {
```

Comment:
Would be nice if I could chain arguments like so; node main.js -dlt test, instead of node main.js -d -l -t test